### DISCUSSION TITLE
Welcome to cof Discussions!

### DISCUSSION THREAD

**Discussion by ZigmundKreud** [ZigmundKreud/cof#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Groslard** [comment]
Bonjour! 
Je me permet de mettre un petit message car je vois que le projet n'a pas bougé depuis juillet.
Est ce qu'il y a toujours du monde qui va bosser dessus ? 
Un portage pour foundry v10 est il prévu ? 

En tout cas, merci pour le travail déjà réalisé !

**Reply by Qaw** [reply]
Bonjour ! Pour avoir les dernières nouvelles n'hésite pas à rejoindre le discord FR La Fonderie : https://discord.gg/F6MSbUPA
Le canal Chroniques Oubliées en particulier ;)
Le projet communautaire est arrêté, car nous travaillons sur une version officielle BBE qui sera compatible V10.

**Comment by Wackamat** [comment]
Bonjour,
J'ai un problème avec le ruleset pour les combats,
j'ai l'impression de "manquer" l'essentiel car mes PJ n'ont rien dans l'onglet "combat" de leur fiche alors que tout les monstre on leurs attaques et on peut même en créer.
C'est peut être lier au fait que je n'arrive pas à équiper une arme a un PJ malgré le fait qu'elle soit dans leur inventaire.
Cependant j'admire le travail que vous avez produits même si je n'arrive pas a le faire fonctionner à son plein potentiel
En attente d'une éventuel réponse
Cordialement

**Reply by Wackamat** [reply]
voici les screens pour mieux comprendre
https://www.zupimages.net/viewer.php?id=23/17/1yi4.jpg
https://zupimages.net/viewer.php?id=23/17/9455.jpg

**Reply by Qaw** [reply]
Pour voir les armes d'un PJ, il faut passer par l'équipement et équiper l'arme avec l'icône du bouclier dans l'onglet inventaire.

**Reply by Wackamat** [reply]
Ha oui effectivement ,merci beaucoup !
Mais est ce qu'il est possible de faire fonctionner les sort de la meme facon ?

**Comment by Qaw** [comment]
Les sorts peuvent être activables.
Mais il faut ensuite les glisser dans la hotbar

**Comment by Yutsa** [comment]
Bonjour,

Je suis développeur et amateur de jeu de rôle. J'aimerais participer au module CoF (et peut-être CoF 2 bientôt ?).

J'ai remarqué en testant un peu le module par exemple que des sorts ne sont pas automatisés. Par exemple lancer le sort "Sous tension" de l'ensorceleur ne déclenche pas de macro pour appliquer l'état "sous tension" et ne propose pas d'ajouter une entré dans la partie "combat" pour lancer la décharge électrique.

Est-ce que vous sauriez me pointer vers une ressource de documentation pour apprendre à écrire les macros nécessaires pour cela ?

J'ai de l'expérience en JS/TS, je ne sais juste pas à quoi j'ai accès dans les macros, par exemple comment je pourrais faire en sorte qu'en lançant le sort "Sous Tenstion" cela applique un effet personnalisé "sous tension" au token ?
